population/dimensions.go

```diff
@@ -24,12 +24,31 @@ type Dimension struct {
 	QualityStatementText string     `json:"quality_statement_text"`
 }
 
+// Slightly confusing given very similar to above, but felt no other way. Remove this comment when others approve naming
+type DimensionCategory struct {
+	Id         string                  `json:"id"`
+	Label      string                  `json:"label"`
+	Categories []DimensionCategoryItem `json:"categories"`
+}
+
+type DimensionCategoryItem struct {
+	ID    string `json:"id"`
+	Label string `json:"label"`
+}
+
 type Category struct {
 	ID                   string `json:"id"`
 	Label                string `json:"label"`
 	QualityStatementText string `json:"quality_statement_text"`
 }
 
+type GetDimensionCategoryInput struct {
+	AuthTokens
+	PaginationParams
+	PopulationType string
+	Dimensions     string
```

Comment:
DImensions? Is this supposed to be singular or a slice?

Reply:
the swagger listed it as a comma separated url query param so i left it as a string here

Reply:
Hmm I think it would be better to have it as a slice for the api client? And convert it to the appropriate string format when creating the URL param. What do we do elsewhere?